pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -60,6 +61,9 @@ func RunClusterInfo(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command) error
 		return err
 	}
 	printService(out, "Kubernetes master", client.Host)
+	if len(client.AlternateHosts) > 0 {
```

Comment:
you mean that AlternateHosts should be merged with Servers when user executes `kubectl config view`?